portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSegments.macro

```diff
@@ -171,6 +171,18 @@ definition {
 		Click(locator1 = "ACSegments#CANCEL_SEGMENT");
 	}
 
+	macro checkDataSourceOrigin {
+		var key_dataSource = "${dataSource}";
+
+		Click(locator1 = "ACSegments#SELECT_BUTTON");
+
+		ACSegments.searchSegment(findSegment = "${documentTitle}");
+
+		AssertElementPresent(locator1 = "ACSegments#DATA_SOURCE_SEGMENT");
+
+		Click(locator1 = "ACSegments#MODAL_CANCEL_BUTTON");
+	}
```

Comment:
Hey @calvinctvo, I liked your macro, but I think this case would only be necessary for this test. What do you think if we put these steps directly into the test?